src/components/Modal/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import MaterialModal from '@material-ui/core/Modal';
+import Backdrop from '@material-ui/core/Backdrop';
+import { createStyles, makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    modal: {
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      position: 'absolute',
+      top: '50%',
+      left: '50%',
+      transform: `translate(-50%, -50%)`,
+    },
+  }),
+);
+
+type ModalProps = {
```

Comment:
How about using interface instead of type alias?
The typescript team recommends always using interfaces unless you cannot express them as interfaces or you need to use union or tuple types.
Please refer to the link below.

https://github.com/Microsoft/TypeScript-Handbook/blob/f728031b7ab1cf54934c86dc41dbf8774369f866/pages/Advanced%20Types.md#interfaces-vs-type-aliases

Reply:
Thank you for your review!
It would be better !